frontend/server/api/auth/customer.post.ts

```diff
@@ -0,0 +1,44 @@
+import { AbortError, ServiceError, fql } from 'fauna'
+
+export default defineEventHandler(async (event) => {
+    // Read customer email from POST in catch all route([...].ts)
+	const body = await readBody(event);
+    const email = body.email;
+
+	// Initialize Fauna client
+	const { client, error } = useFauna()
+	if (error !== null) return error
+
+	try {
+        let customerDetails = null;
+		const query = fql`Customer.customers_by_email(${email})`
+		const response = await client.query(query)
+        //@ts-expect-error
+        const customerData = response.data.data[0]
+        if(response){
+            customerDetails = {
+                id: customerData["id"],
+                email: customerData["email"],
+                firstName: customerData["firstName"],
+                lastName: customerData["lastName"],
+                netWorth: customerData["netWorth"],
+            };
```

Comment:
Keep in mind that the Customer entity now holds a podRef ID. This would have to be updated at a later time. This also applies for any interfaces that were made as a part of this commit